composer.json

```diff
@@ -17,12 +17,12 @@
         }
     ],
     "require": {
-        "php": "^7.1|^8",
+        "php": ">=7.1 <8.2",
         "psr/log": "^1|^2|^3",
         "symfony/var-dumper": "^2.6|^3|^4|^5|^6"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.5.20 || ^9.4.2",
+        "phpunit/phpunit": ">=7.5.20 <10.0",
```

Comment:
Why is this required? We don't need phpunit 8, do we?

Reply:
We don't need PHPUnit 8, but PHPUnit 7 is officially no longer supported, so this puts us on a supported version for older PHP versions.

Reply:
I see unit tests are failing for old PHP versions.  Will fix.  I was unable to test older versions locally due to my machine setup.  Will try to correct that.  Also will clean up temp directory creation.